packages/mui-base/src/useList/useListChangeNotifiers.ts

```diff
@@ -4,58 +4,56 @@ import useMessageBus from '../utils/useMessageBus';
 const SELECTION_CHANGE_TOPIC = 'select:change-selection';
 const HIGHLIGHT_CHANGE_TOPIC = 'select:change-highlight';
 
-export interface SelectChangeNotifiers<TValue> {
+export interface ListChangeNotifiers<ItemValue> {
```

Comment:
I changed all the occurrences of generic parameters such as `T` or `TValue` to more meaningful names without the "T" prefix. We can standardize this approach across the codebase.